### DISCUSSION TITLE
Why the alpha has always an oldest version of react native?

### DISCUSSION THREAD

**Discussion by Pnlvfx** [react-navigation/react-navigation#11847] [General]
I think that if you want more people to test the alpha version you should update react native before releasing the alpha. 
However this is just an idea, it's not a critic.
Just I've tried to use the alpha a few times but I've always noted that is a couple of version behind the latest react-native.

**Comment by satya164** [comment]
Not sure what you mean. The packages don't include any version of React Native and work with any recent versions of React Native.

**Reply by Pnlvfx** [reply]
I get react navigation bottom tabs as an example it has react native 0.73.2 as peerDependencies. (like the other packages of react-navigation)
So if I try to do npm install @react-navigation/bottom-tabs@next with the latest react native version, which is: 0.73.4, I'll get a peer dependencies error and I cannot install it. 
Sorry, I'm not english so somentimes I don't know the right word to use, I hope you understand what I mean.

Bottom tabs peer: 
 "peerDependencies": {
    "@react-navigation/native": "workspace:^",
    "react": "*",
    "react-native": "0.73.2",
    "react-native-safe-area-context": "4.8.2",
    "react-native-screens": "~3.29.0"
  },

**Reply by satya164** [reply]
It's pretty recent, not oldest version. But the hardcoding is an oversight. It should be fixed in latest version https://github.com/react-navigation/react-navigation/commit/4b93b6335ce180fe879f9fbe8f2400426b5484fb

**Reply by Pnlvfx** [reply]
Perfect, thank you
